agent/cmd/cuda-checkpoint-helper/storage_manifest.cpp

```
@@ -0,0 +1,491 @@
+    }
+    jobs->push_back({device_index, extent_index});
+  }
+
+  if (consumed_extents.size() != normalized_extents.size()) {
+    *error = "one or more saved GPU extents were not consumed";
+    return false;
+  }
+  return true;
+}
+
+bool WriteManifest(const std::filesystem::path &directory,
```

Cppcheck (2.21.0)

[style] The function 'WriteManifest' is never used.

(unusedFunction)